services/cron/tasks_extended.go

```diff
@@ -168,7 +168,7 @@ func registerDeleteOldSystemNotices() {
 		OlderThan: 365 * 24 * time.Hour,
 	}, func(ctx context.Context, _ *user_model.User, config Config) error {
 		olderThanConfig := config.(*OlderThanConfig)
-		return system.DeleteOldSystemNotices(olderThanConfig.OlderThan)
+		return system.DeleteOldSystemNotices(ctx, olderThanConfig.OlderThan)
```

Comment:
lines 84 and 85 are still missing

Reply:
You mean I should use `db.DefaultContext` here?

Reply:
No, you should not use `db.DefaultContext` there as it is exactly the same case as here.

Reply:
Instead you should convert `_ context` to `ctx context` and then use this parameter

Reply:
I have made a new commit. I hope it's that what you mean.